Source/Dafny/DafnyOptions.cs

```diff
@@ -1147,6 +1147,8 @@ verification outcome
     1 - Emits a main method in the target language that will execute every method
         in the program with the {{:test}} attribute.
         Cannot be used if the program already contains a main method.
+        Note that /compile:3 or 4 must be provided as well to actually execute
```

Comment:
I'm looking forward to implementing this: https://github.com/dafny-lang/dafny/issues/1322#issuecomment-889023432, which could have a `test` command.

Reply:
I absolutely had that in mind when writing this. :) I've also been meaning to extract that idea into its own issue so it's not just buried in that discussion: https://github.com/dafny-lang/dafny/issues/2036